examples/human.py

```diff
@@ -73,20 +73,20 @@ def userConsole(args):
     while (userInputStr not in exitCommands):
         if (userInputStr == "help"):
             print("Possible actions: ")
-            for actionStr in env.getPossibleActions():
+            for actionStr in env.get_possible_actions():
                 print("\t" + str(actionStr))
 
         elif (userInputStr == "objects"):
             print("Possible objects (one referent listed per object): ")
-            for actionStr in env.getPossibleObjects():
+            for actionStr in env.get_possible_objects():
                 print("\t" + str(actionStr))
 
         elif (userInputStr == "valid"):
             print("Valid action-object combinations:")
-            print(env.getValidActionObjectCombinationsWithTemplates())
+            print(env.get_valid_action_object_combinations_with_templates())
 
         elif (userInputStr == 'goals'):
-            print(env.getGoalProgressStr())
+            print(env.get_goal_progress_srt())
```

Comment:
There's a typo here `..._srt` vs. `..._str`. But fundamentally, do we need it in the first place? I'd be in favor to drop the `_str` all together. The docstring should be clear enough.

Reply:
Oh oops. Somehow that ran on my machine, even with the typo. I'm fine with that. My policy was to just keep the method the calls exactly the same, but I'm happy to change.

Reply:
Done

Reply:
:+1: Just make sure to make the change in scienceworld.py too.

Re: testing, in order to trigger this, you need to run the script and type "goals" as one of the in-game actions.

Reply:
As far as I can tell, get_goal_progress_str is the only method where removing the end type doesn't get rid of any important information, so I think w get rid of just that one.